fa/index.html

```diff
@@ -0,0 +1,142 @@
+    }
+
+    @media screen and (min-width: 400px) {
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>برای پرسیدن سوال نپرس، فقط بپرسش</h1>
+    <p>هر از چند گاهی، توی چت روم هایی که میچرخم، یه نفر هست که چیزی شبیه این میپرسه</p>
+    <blockquote>
+      <span class="name">فلانی:</span>
+      <p class="message">
+          یه جاوا کار حرفه ای هست؟
+      </p>
+    </blockquote>
+    <p>این خوب نیست، به چند دلیل. چیزی که این شخص
+      <em>در واقع</em> میخواد بپرسه اینه که
+    </p>
+    <blockquote>
+      <span class="name">فلانی:</span>
+      <p class="message">
+        آیا یک جاوا کار حرفه ای هست که مشتاق باشه در حل کردن مشکل من بهم کمک کنه، هرچیزی که ممکنه باشه، حتی اگه مشخص بشه سوال ربطی به جاوا نداره و کسی که هیچ چیزی از جاوا نمیدونه هم بتونه به سوالم جواب بده؟
+      </p>
+    </blockquote>
+    <p>
+        تعداد زیادی دلیل وجود داره که افرادی که میدونن جواب نخواهند داد. با این پرسش تو داری خیلی بیشتر از اون چیزی که فکر میکنین تقاضا میکنید
+    </p>
+    <p>
+        تو داری از مردم میخوای که مسئولیت بپذیرن. تو داری اعتماد بنفس اون ها رو در توانایی هاشون زیر سوال میبری. تو داری بدون نیاز بقیه افراد رو از خودت جدا میکنی. من گاهی سوال هایی که مربوط به زبان ها یا کتابخونه هایی که هیچ وقت استفاده نکردم رو جواب میدم، چون که جواب ها (در اگوریتم و مفهوم) جواب یکسانی دارن
+    </p>
+    <p>
+        ...به شکل های دیگه هم میشه دید
+    </p>
+    <blockquote>
+      <span class="name">فلانی:</span>
+      <p class="message">
+        من یک سوال در مورد جاوا دارم اما من خیلی تنبلم برای اینکه سوالم رو به صورت مرتب و قابل فهم بنویسم مگه اینکه یک نفر تو این گروپ باشه که بتونه به سوالم جواب بده
+      </p>
+    </blockquote>
+    <p>
+        که فقط تنبلیه. اگه تو نمیخوای که کاری انجام بدی تا مشکلت حل بشه، چرا ما باید کمک کنیم؟
+    </p>
+    <p>
+        راه حل این مشکل که برای پرسیدن سوال بپرسی، اینه که فقط سوالت رو بپرسی. یک نفر که همش چت هارو میخونه و بیکاره شاید به «سوال برای پرسش» تو جواب بده، ولی اگه همون اول سوالت رو بپرسی توضیح مشکل تو میتونه کنجکاوی اون هار برانگیخته کنه و به جواب سوالت برسی
+    </p>
+    <p>
+        خوب، چکیده اینه که، نپرس
+<em>"جاوا کار حرفه ای هست؟"</em>،
+    بجاش بپرس
+      <em>"چطوری (مشکل) حل کنم؟ با جاوا و (اطلاعات دیگه)"</em>
+    </p>
+    <br>
+    <br>
+    <p>
+      Other similar problems: <a href="http://xyproblem.info/">The XY Problem</a>، <a href="https://www.nohello.com/">No Hello</a>.
+      Further reading: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>،
+      or if you have more time: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+  </main>
+  <footer>
+    Content mostly stolen from <a href="http://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    Translated by painor <a href="https://github.com/painor">@painor</a>
+    -
+    <a href="https://github.com/maunium/dontasktoask.com">Source on Github</a>
```

Comment:
This part should probably be translated too (or at least change the "Translated by" name and link)